README.md

```diff
@@ -42,11 +42,24 @@ Change history can be found in [`HISTORY.md`](https://github.com/mongodb/node-mo
 
 ### Compatibility
 
-For version compatibility matrices, please refer to the following links:
+For server and runtime version compatibility matrices, please refer to the following links:
 
 - [MongoDB](https://www.mongodb.com/docs/drivers/node/current/compatibility/#mongodb-compatibility)
 - [NodeJS](https://www.mongodb.com/docs/drivers/node/current/compatibility/#language-compatibility)
 
+#### Component Support Matrix
+
+The following table describes add-on compponent version compatibility for the Node.js driver. Only packages with versions in these supported ranges are stable when used in combination.
+
+| Component                                                                            | `mongodb@3.x` | `mongodb@4.x`      | `mongodb@5.x`      | `mongodb@6.x` |
+| ------------------------------------------------------------------------------------ | ------------- | ------------------ | ------------------ | ------------- |
+| [bson](https://www.npmjs.com/package/bson)                                           | ^1.0.0        | ^2.0.0             | ^5.0.0             | ^6.0.0        |
```

Comment:
Should we include BSON? There is no configurable compatibility the driver pulls in the version it needs.

Also, iirc, the 4.x driver depended upon bson 4.x.

Reply:
I thought about this, but iirc we had concerns about people potentially installing bson independently and then getting incompatible results. All of you approved https://jira.mongodb.org/browse/WRITING-12378 so it feels like something that would have come up during the review process if we wanted to explicitly exclude js-bson.

Reply:
Yea, I reviewed the google doc comments and didn't see a conversation related to it. I suppose it is informational but potentially confusing to lump it in with "add-on component" so may be worth breaking it out somehow. I don't feel too strongly, resolve if we just want to stick with this as is.

Reply:
I think breaking it out would be even more confusing, so let's stick with this and adjust later if we need to; I fixed the actual versioning